internal/scorecard/plugins/labels_test.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package scplugins
+
+import (
+	"k8s.io/apimachinery/pkg/labels"
+	"testing"
+)
+
+func TestBasicShortNames(t *testing.T) {
+	cases := []struct {
+		name          string
+		testsSelected int
+		wantError     bool
+	}{
+		{"test=" + specBlockExists, 1, false},
+		{"test in (" + specBlockExists + ")", 1, false},
+		{"test in (" + specBlockExists + "," + writeIntoCR + ")", 2, false},
+		{"test=" + statusBlockExists, 1, false},
+		{"test=" + writeIntoCR, 1, false},
+		{"testXwriteintocr", 0, false},
+		{"test X writeintocr", 0, true},
+	}
+
+	for _, c := range cases {
+		t.Run(c.name, func(t *testing.T) {
+			var selector labels.Selector
+			selector, err := labels.Parse(c.name)
+			if err != nil && c.wantError {
+				t.Logf("Wanted error and got error : %v", err)
+				return
+			} else if err != nil && !c.wantError {
+				t.Errorf("Wanted result but got error: %v", err)
+				return
+			}
+
+			basicTests := NewBasicTestSuite(BasicTestConfig{})
+			basicTests.ApplySelector(selector)
+			testsSelected := len(basicTests.Tests)
+			if testsSelected != c.testsSelected {
+				t.Errorf("Wanted testsSelected %d, got: %d", c.testsSelected, testsSelected)
+			}
+		})
+
+	}
+}
+
+func TestOLMShortNames(t *testing.T) {
+	cases := []struct {
+		name          string
+		testsSelected int
+		wantError     bool
```

Comment:
what could be added here is name of the test and expected value. I'd recommend use the IDE to gen the structure since it follows the standard.